.rubocop.yml

```diff
@@ -90,15 +90,27 @@ Style/RedundantRegexpEscape:
 Style/AccessorGrouping:
   Enabled: true
 
-# Disabled due to a bug in Rubocop that raises an exception when accessors
-# are assigned using a splat operator
-# https://github.com/rubocop-hq/rubocop/issues/8257
 Style/BisectedAttrAccessor:
-  Enabled: false
+  Enabled: true
 
 Style/RedundantAssignment:
   Enabled: true
 
+Style/ArrayCoercion:
+  Enabled: true
+
+Style/CaseLikeIf:
+  Enabled: true
+
+Style/HashAsLastArrayItem:
```

Comment:
I wouldn't mind disabling this one, though I don't feel super strongly. Mostly we use this syntax quite a bit in plan language examples, but at the same time the braces do make the code more readable.

Reply:
Disabled 👍